docs/resources/sql_permissions.md

```diff
@@ -5,6 +5,8 @@ subcategory: "Security"
 
 -> **Note** Please switch to [databricks_grants](grants.md) with Unity Catalog to manage data access, which provides better and faster way for managing data security. `databricks_grants` resource *doesn't require a technical cluster to perform operations*. `databricks_sql_permissions` will be removed, once Unity Catalog is Generally Available.
 
+-> **Note** On workspaces with Unity Catalog enabled, you may run into errors such as `Error: cannot create sql permissions: cannot read current grants: For unity catalog, please specify the catalog name explicitly. E.g. SHOW GRANT ``your.address@email.com`` ON CATALOG main`. This happens if your default workspace catalog was set to a UC catalog instead of `hive_metastore`. The workaround is to re-assign the metastore again with the default catalog set to be `hive_metastore`.
```

Comment:
@nkvuong can you also provide that configuration of a single-node cluster with legacy table ACL enabled?

Reply:
the issue is not actually with the cluster config (we still accept the legacy spark conf), it is to do with their default catalog not set to `hive_metastore`

Reply:
oh, okay.